src/NEARWEEK/News.jsx

```diff
@@ -0,0 +1,267 @@
+  gap: 24px;
+  width: 100%;`;
+
+const Card = styled.div`
+    display: flex;
+    @media (max-width: 768px) {
+      display:   ${(props) => (props.index > 4 ? "none" : "flex")}; 
+    }
+`;
+
+const CardContent = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-top: 0;
+    margin-left: 1rem;
+`;
+
+const CardTitle = styled.div`
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    letter-spacing: -0.02em;
+    color: #000000;
+    display: -webkit-box;
+    -webkit-line-clamp: 3;
+    -webkit-box-orient: vertical;  
+    overflow: hidden;
+    a {
+        color: inherit;
+    }
+`;
+
+const CardImage = styled.img`
+    border-radius: var(--bs-border-radius);
+    border: 0.5px solid hsla(210, 12%, 93%, 1);
+    min-width: 78px;
+`;
+
+const CardFooter = styled.div`
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+`;
+
+const CardDate = styled.div`
+    display: flex;
+    align-items: center;
+    gap: 8px;
+    color: hsla(198, 4%, 45%, 1);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 8px;
+    letter-spacing: -0.02em;
+`;
+
+const Badges = styled.div`
+    display: flex;
+    gap: 6px;
+`;
+
+const Badge = styled.span`
+    border: 1px solid;    
+    border-color: hsla(214, 10%, 86%, 1);
+    border-radius: 4px;
+    font-weight: 500;
+    font-size: 8px;
+    line-height: 8px;
+    padding: 2px 4px;
+    text-transform: uppercase;
+    color: hsla(204, 5%, 44%, 1);
+`;
+
+const ButtonLink = styled.a`
+  display: block;
+  width: 100%;
+  padding: 8px;
+  height: 32px;
+  background: #FBFCFD;
+  border: 1px solid #D7DBDF;
+  border-radius: 50px;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 15px;
+  text-align: center;
+  cursor: pointer;
+  color: #11181C !important;
+  margin: 0;
+
+  &:hover,
+  &:focus {
+    background: #ECEDEE;
+    text-decoration: none;
+    outline: none;
+  }
+`;
+
+const news = [...(data?.body.data ?? []), ...posts]
+  .sort((a, b) => new Date(b.createdAt) - new Date(a.createdAt))
+  .slice(0, limit);
+const nwSite = "https://nearweek.com";
+
+function dateToDays(date) {
+  const timeAgo = (diffSec) =>
+    diffSec < 60000
+      ? `${(diffSec / 1000) | 0}s`
+      : diffSec < 3600000
+      ? `${(diffSec / 60000) | 0}m`
+      : diffSec < 86400000
+      ? `${(diffSec / 3600000) | 0}h`
+      : `${(diffSec / 86400000) | 0}d`;
+
+  var d = new Date(date);
+  return timeAgo(Date.now() - d.getTime());
+}
+
+return (
+  <Theme>
+    {data !== null ? (
+      <Wrapper>
+        <H2>News</H2>
+        <Content>
+          {news.map((item, index) => (
+            <Card index={index}>
+              <div class="d-flex">
+                <CardImage width="78" height="78" src={item.thumbnail} alt="" />
+                <CardContent>
+                  <CardTitle>
+                    <a href={article.url} target="_blank">
```

Comment:
@AngelBlock looks like these URLs are not coming through as expected. I'm seeing them open a tab to `about:blank`
<img width="1118" alt="Screen Shot 2023-06-20 at 4 40 54 PM" src="https://github.com/near/near-discovery-components/assets/1915116/5fc93b8d-433f-4285-8390-95a3e085fcc6">

Reply:
Apologies, this is honest mistake, with 4 sources of code, it's hard to maintain. The code was updated to the last known version and PR created #120